HyperVExtension/src/HyperVExtension/Helpers/DevSetupAgentDeploymentHelper.cs

```diff
@@ -128,7 +129,7 @@ private ushort GetVMArchitechture(PSObject session)
 
     private static string LoadScript()
     {
-        var path = Path.Combine(AppContext.BaseDirectory, "HyperVExtension", "Scripts", "DevSetupAgent.ps1");
+        var path = Path.Combine(Package.Current.EffectivePath, "HyperVExtension", "Scripts", "DevSetupAgent.ps1");

```

Comment:
Consider moving `Package.Current.EffectivePath' to a readonly inside a common location.